modular_chomp/maps/overmap/space/fueldepot.dmm

```diff
@@ -1175,13 +1175,9 @@
 /obj/structure/window/reinforced{
 	dir = 1
 	},
+/obj/structure/window/reinforced,
 /obj/structure/window/reinforced{
-	health = 1e+006;
```

Comment:
Before you ask, no, these windows do not serve any purpose and like everywhere else in the code they **should not exist.**

I am removing them and I am not putting them back.